SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -840,7 +840,7 @@ class CEulerSolver : public CFVMFlowSolverBase<CEulerVariable, ENUM_REGIME::COMP
    * \brief Provide the total (inviscid + viscous) non dimensional Equivalent Area coefficient.
    * \return Value of the Equivalent Area coefficient (inviscid + viscous contribution).
    */
-  inline su2double GetTotal_CEquivArea() const final { return Total_CEquivArea; }
+  //inline su2double GetTotal_CEquivArea() const final { return Total_CEquivArea; }
```

Comment:
Are these unused?  If so, can they be removed?

Reply:
I have deleted them.